software/script/chameleon_cli_unit.py

```diff
@@ -1330,6 +1333,7 @@ class HWButtonSettingsSet(DeviceRequiredUnit):
 
     def args_parser(self) -> ArgumentParserNoExit:
         parser = ArgumentParserNoExit()
+        parser.add_argument('-l', '--long', type=int, required=True, help="1 is Long or 0 Short", choices=[0, 1])
```

Comment:
I'd rather see --long as a flag without arg and optional. So short by default unless `--long`.
what do you think?

Reply:
me too.

Reply:
I agree. Done